readme.md

```diff
@@ -127,6 +127,20 @@ Custom promise module to use instead of the native one.
 - Pify has useful options like the ability to handle multiple arguments (`multiArgs`).
 - Pify does not have [magic behavior](https://nodejs.org/api/util.html#util_custom_promisified_functions) for certain Node.js methods and instead focuses on predictability.
 
+#### Why can't a promisified class function access it's own context?
+
+The context of a function can't easily be detected. As a workaround, promisify the whole class and extract the function from the result.
```

Comment:
I would show an example of what's currently not working, and then we have the solution below.

Reply:
Fixed!